### DISCUSSION TITLE
Should the _rowid_ be translatable?

### DISCUSSION THREAD

**Discussion by Stheel83** [sqlitebrowser/sqlitebrowser#3533] [General]
Hi,
when you activate "show rowid column" in the browse data Tab, the column name is not translatable, it stays `_rowid_`
![rowid](https://github.com/sqlitebrowser/sqlitebrowser/assets/153910911/748e0272-17d9-469a-8b4b-f6a72331c88a)

May it be possible to change this behavior? 
Or does this make no sense?

**Comment by chrisjlocke** [comment]
```_rowid_``` is the name of the field, so it doesn't make sense this is translated.  In any language, this should remain ```_rowid_```, the same way the keyword ```select``` or ```from``` is not translated.

**Reply by Stheel83** [reply]
Ok, thanks for the reply.

Then in the programm texts where rowid is mentioned, it should also stay in english. 
And that is the current state in the german translation. 
So, no need to take action here.

Thanks for clarification!
